api/v2/guides/operator-guide-gcp-enclave.md

```diff
@@ -269,3 +269,9 @@ $ gcloud compute instances create uid2-operator-test \
 Note that compared to the `gcloud` command used in the prior section, an additional option
 `--machine-type n2d-standard-16` is added, which ensures production deployment of UID2 Operator runs on
 the recommended machine type that matches the production configuration.
+
+## Upgrading
+
+An email notification will be sent to private operators with an update window. After this window, the old version will be deactivated, and operators on this version will stop functioning.
+This email will include a new cloud-init with the latest version in it. To upgrade you will just need to deploy this cloud-init in the same manor as your orginal operator.
```

Comment:
For each operator version update, private operators receive an email notification with an upgrade window (typically 30 days), after which the old version is deactivated and no longer supported. To upgrade to the latest version, deploy the new `cloud-init` provided in the email in the same manner as the original operator.